makeflow/src/dag_file.c

```diff
@@ -84,4 +87,32 @@ int dag_file_in_trans( const struct dag_file *f )
 		return 0;
 }
 
+int dag_file_mount_clean(struct dag *d) {
```

Comment:
If you would like to free all of the malloced space, then a dag_file_delete function should be written that is not specific to mount. In this function we only partially free malloced space for the dag_file. Also this information is cleaned and removed from the dag_file structures prior to running the workflow, which is not incorrect, but doesn't allow that information to be used be any other parts of makeflow in the future.